src/test/java/guitests/guihandles/PersonListPanelHandle.java

```diff
@@ -394,6 +395,20 @@ public boolean isListMatching(int startPosition, Person... persons) {
         return true;
     }
 
+    public boolean isExactList(Person... persons) {
```

Comment:
Yes. But the implementation of this method is different from mine.  I think for guitest, we should not be grabbing the underlying list from the listview. Comparision should be done between what is shown on the listview by grabbing values from the cardhandle.